frontend/src/app/projects/models/backup-settings/create-backup/create-backup.component.html

```diff
@@ -4,70 +4,101 @@
  -->
 
 <div class="dialog">
-  <h1>Create a new backup pipeline</h1>
-  <form [formGroup]="createBackupForm" *ngIf="t4cAndGitModelExists">
-    <div class="flex flex-row flex-wrap items-center">
-      <div class="collab-card">
-        Please select a T4C model:
-        <mat-selection-list formControlName="t4cmodel" [multiple]="false">
-          <mat-list-option
-            *ngFor="let t4cModel of t4cModelService.t4cModels$ | async"
-            [value]="t4cModel.id"
+  <h2 class="mb-5 text-xl font-medium">Create a new backup pipeline</h2>
+  @if (
+    (this.gitModelService.gitModels$ | async) === undefined ||
+    (this.t4cModelService.t4cModels$ | async) === undefined
+  ) {
+    <ngx-skeleton-loader
+      appearance="circle"
+      [theme]="{
+        'border-radius': '5px',
+        height: '100px',
+        width: '500px',
+        border: '1px solid white',
+      }"
+    ></ngx-skeleton-loader>
+  } @else if (
+    (this.gitModelService.gitModels$ | async)?.length &&
+    (this.t4cModelService.t4cModels$ | async)?.length
+  ) {
+    <form [formGroup]="createBackupForm">
+      <div class="flex flex-row flex-wrap items-center gap-2">
+        <div class="collab-card">
+          Please select a T4C model. <br />
```

Comment:
I think this would look better if it was a bold heading with just "T4C Model". 
The note about having to match could be a little warning card with an icon.

Reply:
![image](https://github.com/user-attachments/assets/74f32454-1e90-401a-a367-8cc164eeb8f2)

That's how it looks now.